multitenancy/tenant-manager-api/src/test/java/io/apicurio/multitenant/RegistryTenantResourceTest.java

```diff
@@ -129,4 +175,19 @@ private void testTenantNotFound(String tenantId) {
            .statusCode(404);
     }
 
+    private static String toString(List<TenantResource> resources) {
+        StringBuilder builder = new StringBuilder();
+        if (resources != null && !resources.isEmpty()) {
+            List<TenantResource> sorted = new ArrayList<>(resources);
+            sorted.sort((r1, r2) -> r1.getType().name().compareTo(r2.getType().name()));
+            sorted.forEach(r -> {
+                builder.append(r.getType().name());
+                builder.append("@");
+                builder.append(r.getLimit());
+                builder.append("\n");
+            });
+        }
+        return builder.toString();
+    }
```

Comment:
@famartinrh I'm wondering if this change removes the need for `equals` and `hashCode` to be implemented on the `TenantResource` class.  Do you know?  Right now, those two methods need to be manually restored whenever we regenerate the jax-rs layer from Studio.  If we didn't need those methods (and I'm thinking with the change above maybe we don't) then codegen would be perfect. :)

Reply:
hmmm, I think you are right, the equals and hashCode where introduced in order for the assertEquals to work. So we can remove them now.